class-plugin-autoupdate-filter.php

```diff
@@ -37,6 +39,57 @@ public function init() {
 		add_filter( 'auto_plugin_update_send_email', '__return_true', 11 );
 		add_filter( 'auto_theme_update_send_email', '__return_true', 11 );
 
+		// "Disable all autoupdates" toggle (killswitch)
+		add_filter( 'auto_update_plugin', array( $this, 'auto_update_killswitch' ), PHP_INT_MAX, 2 );
+		add_filter( 'auto_update_core', array( $this, 'auto_update_killswitch' ), PHP_INT_MAX, 2 );
+		add_filter( 'auto_update_theme', array( $this, 'auto_update_killswitch' ), PHP_INT_MAX, 2 );
+		add_action( 'admin_init', array( $this, 'killswitch_engaged_admin_warning' ) );
+
+	}
+
+	/**
+	 * Load settings from the centralized settings page
+	 */
+	private function get_auto_update_settings() {
+		$endpoint_url = 'https://opsoasis.wpspecialprojects.com/wp-json/custom/v1/get_autoupdate_settings/';
```

Comment:
What do you think of renaming the endpoint: `/wp-json/wpcomsp/v1/autoupdate-settings/`? It's more inline with the REST endpoints defined by the `OpsOasis Endpoints` plugin but it shouldn't matter ultimately.

Reply:
Just thought of `wpcomsp/auto-update-plugin/v1/settings` 😄

The other endpoints are like `wpcomsp/<service>/v1/<resource>`, e.g. `wpcomsp/pressable/v1/sites`, so this would fit in perfectly. Of course, doesn't matter much, just thought I'd bring it up since this would be tricky to change after the first deployment.

Reply:
Valid! Updated endpoint to be `https://opsoasis.wpspecialprojects.com/wp-json/wpcomsp/autoupdate-plugin/v1/settings/`